README.md

```diff
@@ -119,6 +125,8 @@ optional arguments:
   --skip-build          Skip the image build step.
   --force-build         Enforce the image build step, regardless of if the
                         image already is available either locally or remotely.
+  --list-images         print list of images to stdout. Images will not be
+                        built.
```

Comment:
This is not a feature part of this PR, just updated documentation in the README.